src/engine/uct/node/mod.rs

```diff
@@ -146,21 +146,38 @@ impl Node {
                 .iter()
                 .map(|&m| Node::new(m, self.config))
                 .collect();
+            if self.children.len() <= (game.size() * game.size() / 10) as usize {
+                let player = game.last_move().color().opposite();
+                if game.winner() == player || !self.config.uct.playout_aftermath { //don't pass if we're losing on the board
```

Comment:
But we don't call `expand_root()` when we reuse the subtree. So in that case the pass move from the grandchild we're reusing could potentially contain a pass (as you don't check the option in `expand()`).